CMakeLists.txt

```diff
@@ -137,6 +138,9 @@ target_compile_definitions(vcpkglib PUBLIC
     VCPKG_VERSION=${VCPKG_VERSION}
     VCPKG_BASE_VERSION=${VCPKG_BASE_VERSION}
 )
+if (WIN32)
+    target_compile_options(vcpkglib PRIVATE /EHsc)
+endif()
```

Comment:
Best to add `/EHsc` to `CMAKE_CXX_FLAGS`, I think, given that we're adding it to every C++ target; at some point before `add_library`, write `string(APPEND CMAKE_CXX_FLAGS " /EHsc")`